src/milind/DuplicateFromGivenArray.java

```diff
@@ -0,0 +1,44 @@
+package milind;
+
+public class DuplicateFromGivenArray {
+	
+	void duplicateFromIntArray(int[] arr) {
+		System.out.println("Duplicate Element from int array: ");
+		for(int index = 0; index < arr.length; index ++) {
+			int count = 0;
+			for(int index1 = 0; index1 < arr.length; index1 ++) {

```

Comment:
This must be printing 2 times.

Reply:
Do not write logic in main method.
{ 10, 12,32, 55, 32, 17, 12, 32 } try this input. code wont work as expected